backtrace-library/src/main/java/backtraceio/library/models/json/BacktraceAttributes.java

```diff
@@ -164,7 +165,7 @@ private void convertAttributes(BacktraceReport report, Map<String, Object> clien
                 continue;
             }
             Class type = value.getClass();
-            if (type.isPrimitive() || value instanceof String || type.isEnum()) {
+            if (TypeHelper.isPrimitiveOrPrimitiveWrapperOrString(type) || value instanceof String || type.isEnum()) {
```

Comment:
Is `value instanceof String` still needed here? Looks like your new logic will do that too right?

Reply:
Ah good catch. I will remove this check here. Thanks!